dev-image-manifest.aib.yml

```diff
@@ -0,0 +1,140 @@
+name: dev-image
+
+content: 
+  repos:
+    - id: copr-sample-apps
```

Comment:
we need a generalized image without this, and probably without many of the dependencies below as well. Or rather - make it customizable by end-user requirements for buildchain. We should clearly mark our stuff (i.e. tekton,jumpstarter,oc + their deps, mandatory stuff like ssh) and the part that's use-responsibility (boost and whatnot)